Sources/NIO/MarkedCircularBuffer.swift

```diff
@@ -18,29 +18,33 @@
 /// writes and mark how far through the buffer the user has flushed, and therefore how far through the buffer is
 /// safe to write.
 public struct MarkedCircularBuffer<Element>: CustomStringConvertible {
-    private var buffer: CircularBuffer<Element>
-    private var markedIndexOffset: Int? = nil /* nil: nothing marked */
+    @usableFromInline internal var buffer: CircularBuffer<Element>
+    @usableFromInline internal var markedIndexOffset: Int? = nil /* nil: nothing marked */
```

Comment:
Mind prefixing with `_`? We and the stdlib use that convention for ‘should be private’

Reply:
Addressed.